io.openems.edge.common/src/io/openems/edge/common/currency/Currency.java

```diff
@@ -0,0 +1,35 @@
+package io.openems.edge.common.currency;
+
+import io.openems.common.types.OptionsEnum;
+
+public enum Currency implements OptionsEnum {
+	UNDEFINED(-1, "-"), //
+	EUR(0, "€"), //
+	SEK(1, "kr"), //
+	USD(2, "$");
+
+	private final String name;
+	private final int value;
+	public static final Currency DEFAULT = EUR;
```

Comment:
Having UNDEFINED and DEFAULT is irritating.

Move `public static final` to top of the file.